frontend/src/staging/StagingBrowser.vue

```diff
@@ -224,6 +243,13 @@ export default class StagingBrowser extends Vue {
         });
     }
 
+    getFilesToShow(marked:boolean) {
+        if (marked) {
+            return this.filesToShow;
+        }
+        return this.filesToShow.filter(file => !file.marked);
+    }
+
```

Comment:
Der Funktionsparameter ist eigentlich nicht nötig, oder? showMarked ist eine Variable der Komponente, könnte man sich sparen das im Template noch einmal zu übergeben und stattdessen direkt auswerten.

Reply:
Ich gebe Simon recht, das ist nicht notwendig in diesem Fall. Die Komponente ist klein genug und dass die Methode an einer anderen Stelle wiederverwendet wird, ist unwahrscheinlich.